sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -56,22 +78,47 @@ namespace Azure { namespace Core { namespace Http {
      * proxy settings from the system (use no proxy).
      *
      * @remark No validation for the string is done by the Azure SDK. More about this option:
-     * https://curl.haxx.se/libcurl/c/CURLOPT_PROXY.html.
+     * https://curl.se/libcurl/c/CURLOPT_PROXY.html.
      *
      * @remark The default value is an empty string (no proxy).
      *
      */
     Azure::Nullable<std::string> Proxy;
+
+    /**
+     * @brief Username to be used for proxy connections.
+     *
+     * @remark No validation for the string is done by the Azure SDK. More about this option:
+     * https://curl.se/libcurl/c/CURLOPT_PROXYUSERNAME.html.
+     *
+     * @remark The default value is an empty string (no proxy).
+     *
+     */
+    std::string ProxyUsername;
```

Comment:
Is empty the same as null?  Should this be nullable?

Reply:
Answered more thoroughly in a different comment: I couldn't come up with reasonable semantics for an empty `ProxyUserName` that were different from a null proxy user name.